core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -511,6 +513,32 @@ export class DotEditContentHtmlService {
         }
     }
 
+    private injectInlineBlockEditor(): void {
+        const doc = this.getEditPageDocument();
+        const editBlockEditorNodes = doc.querySelectorAll(
+            '.section-blog-post-content[data-content]'
+        );
+        if (editBlockEditorNodes.length) {
+            this.dotLicenseService
+                .isEnterprise()
+                .pipe(
+                    take(1),
+                    filter((isEnterprise: boolean) => isEnterprise === true)
+                )
+                .subscribe(() => {
+                    editBlockEditorNodes.forEach((node) => {
+                        node.classList.add('dotcms__inline-edit-field');
```

Comment:
This can be CSS with a data-attr selector, like we do with containers and contentlets.

Reply:
true !